service/src/test/resources/ehr/request/fhir/output/ExpectedEhrExtractResponseFromJson.xml

```diff
@@ -378,14 +396,18 @@
                                 <center value="20100714175500"/>
                             </effectiveTime>
                             <availabilityTime value="20100714162813"/>
+
                             <pertinentInformation typeCode="PERT">
                                 <sequenceNumber value="+1" />
                                 <pertinentAnnotation classCode="OBS" moodCode="EVN">
                                     <text>This is the BP reading in a consultation with a note</text>
                                 </pertinentAnnotation>
                             </pertinentInformation>
+
+                            <code nullFlavor="UNK"><originalText>Mocked code</originalText></code>
                         </ObservationStatement>
                     </component>
+
```

Comment:
This one and a few xmls above have some random whitespaces, as discusses please investigate